test/e2e/pivoting_test.go

```diff
@@ -24,146 +25,212 @@ import (
 	"sigs.k8s.io/controller-runtime/pkg/client"
 )
 
-const bmoPath = "BMOPATH"
+const (
+	bmoPath                    = "BMOPATH"
+	ironicTLSSetup             = "IRONIC_TLS_SETUP"
+	ironicBasicAuth            = "IRONIC_BASIC_AUTH"
+	KIND                       = "kind"
+	NAMEPREFIX                 = "NAMEPREFIX"
+	restartContainerCertUpdate = "RESTART_CONTAINER_CERTIFICATE_UPDATED"
+	ironicNamespace            = "IRONIC_NAMESPACE"
```

Comment:
The names are a bit confusing (e.g. `bmoPath` is not the actual path to BMO but the variable name). Why are kind and nameprefix all caps?
```suggestion
	bmoPathVar                    = "BMOPATH"
	ironicTLSSetupVar             = "IRONIC_TLS_SETUP"
	ironicBasicAuthVar            = "IRONIC_BASIC_AUTH"
	kind                       = "kind"
	nameprefixVar                 = "NAMEPREFIX"
	restartContainerCertUpdateVar = "RESTART_CONTAINER_CERTIFICATE_UPDATED"
	ironicNamespaceVar            = "IRONIC_NAMESPACE"
```

Reply:
This was the style used by CAPI https://github.com/kubernetes-sigs/cluster-api/blob/6c6543448bee351430e48f39883265f409c44662/test/e2e/common.go#L37-L47

Reply:
Ok I see, let's keep the names as is then. But what about KIND and NAMEPREFIX? Why not change to `kind` and `nameprefix`?

Reply:
fixed